apps/visitor-app/backend/modules/database/db_functions.bal

```diff
@@ -71,32 +71,63 @@ public isolated function addInvitation(AddInvitationPayload payload, string crea
 #
 # + encodeValue - Encoded uuid value
 # + return - Invitation object or error
-public isolated function fetchInvitation(string encodeValue) returns Invitation|error {
-    Invitation|sql:Error invitation = databaseClient->queryRow(fetchInvitationQuery(encodeValue));
-    if invitation is sql:Error {
-        return invitation;
+public isolated function fetchInvitation(string encodeValue) returns Invitation|error? {
+    InvitationRecord|error invitationRecord = databaseClient->queryRow(fetchInvitationQuery(encodeValue));
+    if invitationRecord is error {
+        if invitationRecord is sql:NoRowsError {
+            return;
+        }
+        return invitationRecord;
```

Comment:
```suggestion
    return invitationRecord is sql:NoRowsError ? () : invitationRecord;
```

Reply:
Updated as follows `if visitor is error {
        return visitor is sql:NoRowsError ? () : visitor;
    }` . Need to process the record after.